src/routes/transactions/__tests__/controllers/add-transaction-confirmations.transactions.controller.spec.ts

```diff
@@ -87,7 +87,7 @@ describe('Add transaction confirmations - Transactions Controller (Unit)', () =>
         .with('name', faker.word.words())
         .build(),
     ];
-    const replacementTxsPage = pageBuilder().with('results', []).build();
+    const rejectionTxsPage = pageBuilder().with('results', []).build();
```

Comment:
I've just renamed `replacement*` to `rejection*` in this file, I think it's more meaningful and makes the logic easier to follow.